docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -6824,3 +6824,36 @@ function mass_theme_preprocess_paragraph__organization_contact_logo(&$variables)
   $variables['org_logo'] = $node->get('field_sub_brand')->view('logo_link');
 
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Break out url pieces for callout link paragraphs.
+ */
+function mass_theme_preprocess_paragraph__org_section_long_form(&$variables) {
+  $components = [];
+  $container = $variables['paragraph'];
+  $paragraphs = Helper::getReferencedEntitiesFromField($container, "field_section_long_form_content");
+
+  for ($i = 0; $i < count($paragraphs); $i++) {
```

Comment:
@Javi-er Is there a particular reason that for was chosen here rather than a foreach? Specifically, were there any performance-related issues worth noting?

Reply:
@tom-fleming no, there weren't any performance issue, I'm using a for instead of foreach because the index is needed for some operations below, specifically the index is moved ahead one position if the current paragraph needs wrapping.